.github/workflows/pypi-nightly-build.yml

```diff
@@ -52,15 +52,104 @@ jobs:
           --wheel
           --outdir dist/
           .
+
+      # Trigger Buildkite smoke tests
+      - name: Trigger Buildkite Smoke Tests
+        id: trigger_buildkite
+        env:
+          BUILDKITE_TOKEN: ${{ secrets.BUILDKITE_TOKEN }}
+        run: |
+          data='{
+            "commit": "${{ github.sha }}",
+            "branch": "master",
+            "message": "nightly-build-pypi",
+            "ignore_pipeline_branch_filters": true,
+            "env": {
+              "ARGS": "--aws"
+            }
+          }'
+
+          response=$(curl -w "%{http_code}" -H "Authorization: Bearer $BUILDKITE_TOKEN" \
+            -X POST "https://api.buildkite.com/v2/organizations/skypilot-1/pipelines/smoke-tests/builds" \
+            -d "$data")
+
+          http_code=$(echo "$response" | tail -n 1)
+          response_body=$(echo "$response" | head -n -1)
+
+          if [ "$http_code" != "201" ]; then
+            echo "Error: Buildkite API returned HTTP status code $http_code"
+            echo "Response body:"
+            echo "$response_body"
+            exit 1
+          fi
+
+          build_url=$(echo "$response_body" | jq -r '.web_url')
+          build_number=$(echo "$response_body" | jq -r '.number')
+
+          echo "Build triggered successfully (HTTP status code: $http_code)"
+          echo "Build URL: $build_url"
+          echo "::set-output name=build_url::$build_url"
+          echo "::set-output name=build_number::$build_number"
```

Comment:
Can we just use the existing GH action we have? Check here for calling other workflows: https://github.com/skypilot-org/skypilot/blob/c5039370280815a3f347e76622dc154ede36d6c3/.github/workflows/helm-docker-release.yaml#L27-L34